PowervRA/Functions/Public/config-service/Connect-vRAVAMI.psm1

```diff
@@ -0,0 +1,130 @@
+function Connect-vRAVAMI {
+<#
+    .SYNOPSIS
+    Connect to a vRA Appliance
+    
+    .DESCRIPTION
+    Connect to a vRA Appliance and generate a connection object with Servername, Token etc
+    
+    .PARAMETER Appliance
+    vRA Appliance to connect to
+
+    .PARAMETER Username
+    Username to connect with
+
+    .PARAMETER Password
+    Password to connect with
+
+    .PARAMETER Credential
+    Credential object to connect with
+
+    .PARAMETER IgnoreCertRequirements
+    Ignore requirements to use fully signed certificates
+
+    .INPUTS
+    System.String
+    Management.Automation.PSCredential
+    Switch
+
+    .OUTPUTS
+    System.Management.Automation.PSObject.
+
+    .EXAMPLE
+    Connect-vRAAppliance -Appliance vraappliance01.domain.local -Username root -Password P@ssword -IgnoreCertRequirements
+
+    .EXAMPLE
+    Connect-vRAAppliance -Appliance vraappliance01.domain.local -Credential (Get-Credential)
```

Comment:
The name of the function Connect-vRAVAMI doesn't match what is used in the examples, Connect-vRAAppliance